src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -159,7 +159,9 @@ class ComposeUp extends DefaultTask {
         logger.info("Will use $host as host of service $serviceName")
         def tcpPorts = settings.dockerExecutor.getTcpPortsMapping(serviceName, inspection, host)
         def udpPorts = settings.dockerExecutor.getUdpPortsMapping(serviceName, inspection, host)
-        String instanceName = inspection.Name.find(/${serviceName}_\d+/) ?: inspection.Name - '/'
+        String instanceName = inspection.Name.find(/${serviceName}_\d+$/) ?:
```

Comment:
Could you please add a comment explaining what's happening here? 🙏

Reply:
Of course.  👍